backend/src/test/java/com/github/astridottmann/backend/services/CalculateCo2EmissionServiceTest.java

```diff
@@ -10,10 +11,10 @@ class CalculateCo2EmissionServiceTest {
 CalculateCo2EmissionService calculateCo2EmissionService = new CalculateCo2EmissionService();
     @Test
     void calculateCo2EmissionRoute() {
-        Vehicle vehicle = new Vehicle("Car", 120);
+        Vehicle vehicle = new Car("car", 160, "petrol", "small" );
         RouteDTO route = new RouteDTO("Berlin", "Potsdam", 100, 2,true, vehicle);
 
-        double expectedCo2Emission = vehicle.getCo2Emission() * 100 / 2 / 1000;
+        double expectedCo2Emission = route.vehicle().getCo2Emission() * 100 / 2 / 1000;
```

Comment:
maybe it makes sense when you add a CalculatorService. So you do not have to type the magic numbers for the expected emission.